tests/e2e/common/kubernetes.go

```diff
@@ -213,3 +215,18 @@ func cleanupNodeFeatureRules(ctx context.Context, cli *nfdclient.Clientset) {
 		}
 	}
 }
+
+// CreateKubernetesClient creates a Kubernetes clientset from the given kubeconfig file
+func CreateKubernetesClient(kubeconfig string) (*kubernetes.Clientset, error) {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
```

Comment:
Is this the recommended way to do this? We already construct a client in the `internal/flags` package. Should we reuse that for the tests, or are there concerns with portability?